generated/gitops-template/githubactions/.github/workflows/gitops-promotion.yml

```diff
@@ -25,6 +25,8 @@ env:
   TRUSTIFICATION_OIDC_CLIENT_ID: ${{ secrets.TRUSTIFICATION_OIDC_CLIENT_ID }}
   TRUSTIFICATION_OIDC_CLIENT_SECRET: ${{ secrets.TRUSTIFICATION_OIDC_CLIENT_SECRET }}
   TRUSTIFICATION_SUPPORTED_CYCLONEDX_VERSION: ${{ secrets.TRUSTIFICATION_SUPPORTED_CYCLONEDX_VERSION }}
+  QUAY_IO_CREDS_USR: ${{ secrets.QUAY_IO_CREDS_USR }}
```

Comment:
Why only quay credential? We also have other private registries like for artifactory/nexus

Reply:
Honestly I'm not sure about that either, but I think clearing that up is out of scope for this PR.